src/cascadia/QueryExtension/ExtensionPalette.idl

```diff
@@ -36,7 +36,16 @@ namespace Microsoft.Terminal.Query.Extension
         Windows.UI.Xaml.Controls.IconElement ResolvedIcon { get; };
 
         event Windows.Foundation.TypedEventHandler<ExtensionPalette, IInspectable> ActiveControlInfoRequested;
-        event Windows.Foundation.TypedEventHandler<ExtensionPalette, IInspectable> AIKeyAndEndpointRequested;
         event Windows.Foundation.TypedEventHandler<ExtensionPalette, String> InputSuggestionRequested;
     }
+
+    [default_interface] runtimeclass TerminalContext : IContext
+    {
+        TerminalContext(String activeCommandline);
+    }
+
+    [default_interface] runtimeclass SystemResponse : IResponse
```

Comment:
We don't need `[default_interface]` here. It's incorrect to have it unless we are going to refer to an `ISystemResponse`.